include/boost/url/impl/url_base.ipp

```diff
@@ -1725,8 +1725,7 @@ set_encoded_fragment(string_view s)
     s = buf.maybe_copy(s);
     check_invariants();
     auto t = grammar::parse(
-        s, detail::fragment_rule).value(
-            BOOST_CURRENT_LOCATION);
+        s, detail::fragment_rule).value(BOOST_URL_POS);
```

Comment:
do we even need to pass something to `value`? It already has a default parameter with the source location.